### ISSUE TITLE
missing `$` at the beginning of the environment variables

### ISSUE THREAD

**Issue by ganezasan** [CircleCI-Public/aws-ecs-orb#9]
If we put some `aws` properties as below script, it doesn't set `$` at the beginning of the environment variables.  It seems [aws-ecr/build_and_push_image](https://github.com/CircleCI-Public/aws-ecr-orb) can set the environment variables without `$`. I think it's better to be able to set environment variables the same way.

```
      - aws-ecs/deploy-service-update:
          requires:
            - aws-ecr/build_and_push_image
          aws-access-key-id: AWS_ACCESS_KEY
          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
          aws-region: AWS_REGION
```
↓
```
    - run:
        name: Configure AWS Access Key ID
        command: |
          aws configure set aws_access_key_id \
          AWS_ACCESS_KEY \
          --profile default
    - run:
        name: Configure AWS Secret Access Key
        command: |
          aws configure set aws_secret_access_key \
          AWS_SECRET_ACCESS_KEY \
          --profile default
    - run:
        name: Configure AWS default region
        command: |
          aws configure set region AWS_REGION \
          --profile default
```

**Comment by lokst** [comment]
Thanks @ganezasan , I'll look into this!

**Comment by PseudoNerd** [comment]
Can I take this up?

**Comment by lokst** [comment]
@PseudoNerd Sure, do give it a go!

**Comment by sameshl** [comment]
@ganezasan I would love to take this up. Could you tell where exactly this change needs to be done?
Thanks!

**Comment by KyleTryon** [comment]
This will be addressed as part of #118
